build.gradle

```diff
@@ -93,10 +93,10 @@ subprojects { project ->
                 testImplementation "org.spockframework:spock-core:${spockVersion}", {
                     exclude module: 'groovy-all'
                 }
-                testImplementation "cglib:cglib-nodep:2.2.2"
-                testImplementation "org.objenesis:objenesis:1.4"
+                testImplementation "net.bytebuddy:byte-buddy:$bytebuddyVersion"
+                testImplementation "org.objenesis:objenesis:$objenesisVersion"
```

Comment:
I think these can narrowed to `testRuntimeOnly`

Reply:
I have made these changes